<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicWindow</class>
 <widget class="QMainWindow" name="MusicWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-weight:600; font-style:italic;&quot;&gt;Add your song!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="SongTable">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>661</width>
      <height>341</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>129</width>
      <height>178</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="ArtistLine">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Artist</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="SongLine">
         <property name="placeholderText">
          <string>Song</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="GenreLine">
         <property name="placeholderText">
          <string>Genre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DescriptionLine">
         <property name="placeholderText">
          <string>A word to describe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LinkLine">
         <property name="placeholderText">
          <string>Link</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="addSongButton">
       <property name="text">
        <string>ADD SONG</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>70</y>
      <width>120</width>
      <height>341</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>118</width>
       <height>339</height>
      </rect>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>5</y>
        <width>91</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="CheckRock">
         <property name="text">
          <string>Rock</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckIndie">
         <property name="text">
          <string>Indie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckHip">
         <property name="text">
          <string>Hip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckRap">
         <property name="text">
          <string>Rap</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckEDM">
         <property name="text">
          <string>EDM</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
